docs/framework/unmanaged-api/debugging/icordebugcode-getsize-method.md

```diff
@@ -32,16 +32,15 @@ HRESULT GetSize (
 
 ## Parameters
 
- `pcBytes`  
- [out] A pointer to the size, in bytes, of the binary code that this `ICorDebugCode` object represents.
+`pcBytes` \
```

Comment:
we've stopped adding the \ and went back to two spaces because it caused issues for loc in some cases

```suggestion
`pcBytes`  
```

Reply:
OK, did you look at using H3s or definition lists? The space as formatting is kind of fragile (also semantically not great)

Reply:
Yeah I know it's fragile 😢 haven't looked at other formatting for these yet.